ocaml/xapi/xapi_host.ml

```diff
@@ -2701,14 +2706,65 @@ let write_uefi_certificates_to_disk ~__context ~host:_ =
     | Error _ ->
         debug
           "UEFI tar file was not extracted: it was not base64-encoded correctly"
+  )
+
+let write_uefi_certificates_to_disk ~__context ~host =
+  let with_valid_symlink from_path to_path fn =
```

Comment:
Since the order of arguments here is the opposite of the order in Unix.symlink perhaps use labeled arguments here to make it obvious at the callsite which argument means what.